GNSSLogger/app/src/main/java/com/google/android/apps/location/gps/gnsslogger/SensorFusionContainer.java

```diff
@@ -21,19 +21,21 @@
 import android.location.GnssNavigationMessage;
 import android.location.GnssStatus;
 import android.location.Location;
-import android.location.LocationListener;
 import android.location.LocationManager;
 import android.location.OnNmeaMessageListener;
 import android.os.Bundle;
 import android.os.SystemClock;
 import java.util.Arrays;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
+import com.google.android.gms.common.api.GoogleApiClient;
+import com.google.android.gms.location.LocationRequest;
+import com.google.android.gms.location.LocationServices;
 
 /**
  * A container for GPS related API calls, it binds the {@link LocationManager} with {@link UiLogger}
  */
-public class GnssContainer {
+public class SensorFusionContainer {
```

Comment:
I think this class can have a more appropriate name. What this class appears to do is provide measurements from a few measurement providers - GNSS, sensor, fused - to an array of listeners. Why not call it something along the lines of `MeasurementProvider`? Also please update the javadoc to reflect that this class binds with many providers, not just GNSS anymore, and that the listeners are any instances of `MeasurementListener`, not just `UiLogger`.

Reply:
I struggled with what to call this class as well. Thanks for the suggestion, it's been implemented!